src/mixins/object_geometry.mixin.js

```diff
@@ -88,8 +88,15 @@
      * The coords are returned in an array.
      * @return {Array} [tl, tr, br, bl] of points
      */
-    getCoords: function(absolute, calculate) {
-      return arrayFromCoords(this._getCoords(absolute, calculate));
+    getCoords: function (absolute, calculate) {
+      var coords = arrayFromCoords(this._getCoords(absolute, calculate));
+      if (this.group) {
+        var t = this.group.calcTransformMatrix();
+        return coords.map(function (p) {
+          return util.transformPoint(p, t);
+        });
+      }
+      return coords;
```

Comment:
the issue here is that if now you want to do some check in group coordinates, you can't anymore.
Is also true that maybe that is not useful.

Reply:
I am thinking of a method in `util`. Something like 
- `transformPointBetweenPlanes(from, to)`
- `getPointRelativeTo(originObject, destinationObject)` - here is it confusing because it can be relative to destination as a sibling or as a child.

Reply:
- `transformPointBetweenPlanes(point, originObject, destinationObject, originPlaneRelativity, destinationPlaneRelativity)`
`originPlaneRelativity` can be a boolean indicating if the point is a sibling or a child in relation to object

Reply:
It is a good util that will probably be useful in fabric and for devs

Reply:
#7614